src/components/chips/js/chipsController.js

```diff
@@ -50,6 +51,26 @@ function MdChipsCtrl ($scope, $mdConstant, $log, $element, $timeout) {
   /** @type {boolean} */
   this.hasAutocomplete = false;
 
+  /** @type {string} */
+  this.$scope.enableChipEdit = $attrs['mdEnableChipEdit'] || '';
```

Comment:
This variable should be automatically injected to scope. So just use the variable `$scope.mdEnableChipEdit`, it's already interpolated.

Use it like so:
```js
$scope.mdEnableChipEdit = $mdUtil.parseAttributeValue($scope.mdEnableChipEdit);
```